dalite_xblock/dalite_xblock.py

```diff
@@ -143,6 +153,30 @@ def is_lti_ready(self):
         """Check if this XBlock has all settings so it can connect to the LTI."""
         return all((self.launch_url, self.question_id, self.assignment_id))
 
+    def _render_student_view(self, context, in_studio):
+
+        if in_studio:
+            message = _("No question selected. Please click \"Edit\" and enter the assignment ID and question ID.")
+        else:
+            message = _("This task is not configured yet. You can't finish it right now.")
+
```

Comment:
This message is a bit blunt..  And students will only see this message when the question has been published, but not properly configured.

Maybe it's better to say: "This question is not configured yet. Please tell your instructor if you see this error."